ec2/InstanceType.dhall

```diff
@@ -3,7 +3,14 @@ let imports = ../imports.dhall
 let Prelude = imports.Prelude
 
 let InstanceType =
-      < c5-large
+      < t3-nano
+      | t3-micro
+      | t3-small
+      | t3-medium
+      | t3-large
+      | t3-xlarge
+      | t3-2xlarge
```

Comment:
This list was maintained in alphabetical order for the instance family - please move t3 to the end